src/vis/vis.js

```diff
@@ -424,14 +423,26 @@ var Vis = View.extend({
     });
 
     // Trigger 'done' event
-
     _.defer(function () {
       self.trigger('done', self, self.map.layers);
     });
 
+    if (!options.skipMapInstantiation) {
+      this.instantiateMap();
+    }
+
     return this;
   },
 
+  instantiateMap: function (invalidateMapSize) {
```

Comment:
This argument should not really be necessary, see #1105 which contains the proposed change